server/controller/src/main/java/ai/starwhale/mlops/domain/task/status/watchers/log/TaskLogK8sCollector.java

```diff
@@ -60,8 +60,7 @@ public void collect(Task task) throws StarwhaleException {
         try {
             taskLog = k8sClient.logOfJob(K8sClient.toV1LabelSelector(Map.of(
                             K8sJobTemplate.JOB_IDENTITY_LABEL, task.getId().toString())),
-                    Stream.concat(k8sJobTemplate.getInitContainerTemplates().stream(),
-                                    k8sJobTemplate.getContainersTemplates().stream())
```

Comment:
just delete init containers in the template works too.

Reply:
done